google/cloud/storage/client.h

```diff
@@ -68,6 +91,18 @@ class Client {
     return raw_client_;
   }
 
+  //@{
+  /**
+   * @name Bucket operations.
+   *
+   * Buckets are the basic containers that hold your data. Everything that you
```

Comment:
s/Everything/Every object/

Reply:
But also notifications are associated with buckets, and the wording is verbatim from here:

https://cloud.google.com/storage/docs/key-terms#buckets

Reply:
Seems reasonable enough, and it's consistent with other wording, so LGTM :)